src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <div>
+      <h4>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></h4>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-owns="id"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
+      <ul
+        :aria-label="selectLabel"
+        :class="getUlClass()"
```

Comment:
This seems like a good place for a computed class

Reply:
is the function a no no?